common/persistence/sql/sqlTaskManager.go

```diff
@@ -264,6 +264,9 @@ type taskListPageToken struct {
 	TaskType int64
 }
 
+// ListTaskList lists tasklist from DB
+// DomainID translates into byte array in SQL. The minUUID is not the minimum byte array.
+// This API could return incomplete result set.
```

Comment:
Shall we create an issue for this? Looks like this is something we need to fix.

Reply:
https://github.com/uber/cadence/issues/3911